### PR TITLE
Support to PostgreSQL 12

### PR DESCRIPTION
**Pull request by MoonInsung**
#124 
This is a pg_rman patch for postgresql12.

The list of change list is as follows.
1. From PG12, recovery-related options are moved from recover.conf to postgresql.conf.
These parts have been fixed.
2. Files such as recovery.signal are required for recovery.
These parts have been fixed.
3. The argument of the get_controlfile function has been changed from PG12.
These parts have been fixed.
4. Changed some strings for regression tests.

### CODE REVIEW

**Comment by MoonInsung** [comment]
It is need to fix it related to remote backup ...
I will correct this part at a later date.

**Comment by MoonInsung** [comment]
Passed the regression test in my environment.
maybe..done..

File: backup.c [hunk 1]
```diff
@@ -856,12 +856,16 @@ do_backup(pgBackupOption bkupopt)
...
 			(errcode(ERROR_ARGS),
 			 errmsg("required parameter not specified: SRVLOG_PATH (-S, --srvlog-path)")));
-
 	/*
-	 * If we are taking backup from standby (ie, $PGDATA has recovery.conf),
+	 * If we are taking backup from standby
+	 * (ie, $PGDATA has recovery.conf or standby.signal),
 	 * check required parameters (ie, standby connection info).
 	 */
+#if PG_VERSION_NUM >= 120000
```
[the remark this answers is not in the document]
Comment (amitlan) [reply backup.c]: Moon-san, it appears I forgot to push some button, so this comment wasn't sent to you yesterday.  Anyway, I will fix it myself.

**Comment by amitlan** [comment]
Thank you Moon-san, I have merged this PR for now.  Will fix if there's anything to be fixed.

**Comment by dostrow** [comment]
Something strange happened in your merge, the checks for the version around the get_controlfile argument changes seem to have been dropped. Can you have a look?

**Pull request merged**